ordered_associative_task/student/include/impl.h

```diff
@@ -15,4 +16,15 @@
  *
  * @return 1-5
  */
-size_t convertMarkTo5PointSystem(size_t markIn100Point);
+
+std::map<size_t, size_t> markLimits {
+    {29, 1},
+    {49, 2},
+    {69, 3},
+    {89, 4},
+    {100, 5}
+};
+
+size_t convertMarkTo5PointSystem(size_t markIn100Point) {
+    return markLimits.lower_bound(std::min(markIn100Point, (size_t)100))->second;
```

Comment:
hacky way, try to solve it with such mapping:
{{0, 1}, {30, 2}, {50, 3}, {70, 4}, {90, 5}};

Also, why should you cast 100?
use for constants https://en.cppreference.com/w/cpp/language/integer_literal

Reply:
I casted 100 because std::min need arguments of the same type. I resolved the both problems.